src/main/scala/uk/gov/nationalarchives/tdr/api/service/FinalTransferConfirmationService.scala

```diff
@@ -42,7 +42,7 @@ class FinalTransferConfirmationService(
         Seq(
           ConsignmentmetadataRow(uuidSource.uuid, standard.consignmentId, LegalCustodyTransferConfirmed, standard.legalCustodyTransferConfirmed.toString, time, userId)
         )
-      case judgment: AddFinalJudgmentTransferConfirmationInput =>
+      case judgment: AddFinalTransferConfirmationInput =>
```

Comment:
Since the standard & judgments are essentially the same now. You can refactor the method to something like this
``` 
private def convertInputToPropertyRows(inputs: AddFinalTransferConfirmationInput, userId: UUID): Seq[ConsignmentmetadataRow] = {
    val time = Timestamp.from(timeSource.now)
    Seq(ConsignmentmetadataRow(uuidSource.uuid, inputs.consignmentId, LegalCustodyTransferConfirmed, inputs.legalCustodyTransferConfirmed.toString, time, userId))
  }
```